Marlin/src/HAL/STM32/tft/tft_fsmc.cpp

```diff
@@ -74,8 +74,8 @@ void TFT_FSMC::Init() {
   Timing.AddressHoldTime = 15;
   Timing.DataSetupTime = 24;
   Timing.BusTurnAroundDuration = 0;
-  Timing.CLKDivision = 16;
-  Timing.DataLatency = 17;
+  Timing.CLKDivision = TERN(TFT_INTERFACE_FSMC_8BIT, 0, 16);
+  Timing.DataLatency = TERN(TFT_INTERFACE_FSMC_8BIT, 0, 17);
   Timing.AccessMode = FSMC_ACCESS_MODE_A;
```

Comment:
@thinkyhead: @rhapsodyv had me revert this change and there was no difference in how the TFT35 responded. It felt just as snappy with either change, but he said that 0 is an invalid value.

Does your TFT28 act any differently between these changes?